doc/api/http.md

```diff
@@ -744,6 +744,39 @@ added: v0.4.0
 
 * {string} The request path.
 
+### `request.method`
+<!-- YAML
+added: REPLACEME
+changes:
```

Comment:
Generally these `changes` sections are only needed when the *code* is modified *after* it's added, which is not the case for any of these.